Library/DasharoSystemFeaturesUiLib/DasharoSystemFeatures.c

```diff
@@ -993,6 +1019,26 @@ DasharoSystemFeaturesRouteConfig (
     }
   }
 
+  if(DasharoFeaturesData.BatteryConfig.StartThreshold > DasharoFeaturesData.BatteryConfig.StopThreshold) {
+    return EFI_INVALID_PARAMETER;
+  }

```

Comment:
It should be rather done in callback and create a pop-up window when user enters new values. If we do it here, don't know what will happen to other changed options if you change multiple at once. Have you tested what happens if you actually trigger this incorrect configuration?